components/config-mgmt-service/backend/client.go

```diff
@@ -17,8 +17,8 @@ import (
 // use in the future. Think about it as a set of contracts that the API
 // requires to fulfill the requests
 type Client interface {
-	// @params (nodeid)
-	GetNode(string) (Node, error)
+	// @params (nodeid, filters)
+	NodeExists(nodeID string, projectFilters map[string][]string) (bool, error)
```

Comment:
The `GetNode` function was only being used by the `GetRuns` function to see if the node existed.